### DISCUSSION TITLE
SOLARIS-HEPPA-CMIP-4-Y: solar forcing

### DISCUSSION THREAD

**Discussion by vnaik60** [PCMDI/input4MIPs_CVs#19] [General]
Just highlighting that Doug Kinnison from NCAR/ACOM evaluated the [preliminary version of the solar forcing](https://solarisheppa.geomar.de/cmip7) using the WACCM6 model. Presentation is available [here ](https://www.youtube.com/watch?v=yHcCZksa4-c). Ping @durack1 @berndfunke. 

EDIT: note that the dataset that NCAR evaluated was a far earlier version that SOLARIS-HEPPA-CMIP-4-4

**Comment by durack1** [comment]
Nice @vnaik60 as it's a 9hr recording, is there an agenda that can more tightly target the presentation? Seems like the video doesn't have sections identified yet either.

Got it, starts at 5:31 - https://www.youtube.com/live/yHcCZksa4-c?si=9q2TqYk6LTvyDDdY&t=19913

**Comment by vnaik60** [comment]
https://www.cesm.ucar.edu/events/351/agenda here it is now updated with
links etc.

On Saturday, June 15, 2024, Paul J. Durack ***@***.***> wrote:

> Nice @vnaik60 <https://github.com/vnaik60> as it's a 9hr recording, is
> there an agenda that can more tightly target the presentation? Seems like
> the video doesn't have sections identified yet either
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/PCMDI/input4MIPs_CVs/discussions/19#discussioncomment-9782524>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANKNY5G3TNNULCX7K5Q46NDZHROWZAVCNFSM6AAAAABJKRNPOWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TOOBSGUZDI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Dr. Vaishali Naik <http://gfdl.noaa.gov/vaishali-naik-s-homepage> (she/her)
Physical Scientist
DOC/NOAA/OAR/Geophysical Fluid Dynamics Laboratory
Earth System Processes and Interactions Division
201 Forrestal Rd.
Princeton, NJ 08540
Phone: (609) 987 5057
Fax: (609) 987 5063

**Comment by durack1** [comment]
@st-bender @bfunke - we are trying to centralize discussions here.. No comments from modelling groups, yet..

**Comment by znicholls** [comment]
@st-bender @bfunke a question I received from one of the European modelling teams (Roland Seferian): we note a clear difference compared to CMIP6. It is really important for the preindustrial period. Do you have guidance on how to integrate this information in ESMs ?

Any thoughts or advice?

**Reply by znicholls** [reply]
@ Roland if you're looking at this, @vnaik60 has some info:

> Bernd et al have documented the difference in methodology between CMIp6 and CMIP6plus in the [paper] (https://gmd.copernicus.org/articles/17/1217/2024/), on their [website ](https://solarisheppa.geomar.de/cmip7), which is also reflected [here](https://input4mips-controlled-vocabularies-cvs.readthedocs.io/en/latest/dataset-overviews/solar/). Perhaps Roland has a specific question relating to ESMs, which may be helpful for other modellers too though

**Reply by st-bender** [reply]
Hi,
I am not sure what the issue is, "clear differences" is quite vague.

There are differences in SSI and TSI forcing, as described on the webpage and in the referenced paper.
We tried to publish the changed radiation forcing early in the process, so that groups can adapt and start tuning early. SSI and TSI have been "stable" since version 4.4 from October 2024.
Cheers.
